src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
```

Comment:
```suggestion
* {
  box-sizing: border-box;
```
![image](https://user-images.githubusercontent.com/96409563/236266640-053454f8-98de-476b-b872-ac2c64b5d0cd.png)